config/routes.rb

```diff
@@ -218,9 +218,6 @@
         get '/missing' => 'work_history/explanation#show', as: :work_history_explanation
         post '/missing' => 'work_history/explanation#submit'
 
-        get '/explain-breaks' => 'work_history/breaks#edit', as: :work_history_breaks
```

Comment:
Spotted a bug while testing https://github.com/DFE-Digital/apply-for-teacher-training/pull/4090, but the comment probably fits better in this PR.

This line still uses the path helper deleted above. https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/components/candidate_interface/work_history_review_component.rb#L45

Reply:
Good spot. So this only surfaces when an application that used the first implementation is accessed via the review previous applications feature. So it still needs to exist in terms of being able to render/view, but will never need the change links. It would have broken the page though.